lib/trino-parquet/src/main/java/io/trino/parquet/ParquetReaderOptions.java

```diff
@@ -73,6 +73,21 @@ private ParquetReaderOptions(
         this.vectorizedDecodingEnabled = vectorizedDecodingEnabled;
     }
 
+    public static Builder builder()
+    {
+        return new Builder(new ParquetReaderOptions());
+    }
+
+    public static Builder builder(ParquetReaderOptions parquetReaderOptions)
+    {
+        return new Builder(parquetReaderOptions);
+    }
+
+    public static ParquetReaderOptions defaultOptions()
+    {
+        return new ParquetReaderOptions();
+    }
```

Comment:
remove/inline this method, in favour of `ParquetReaderOptions.builder().build()`